KarhooUISDK.xcodeproj/xcshareddata/xcschemes/Client+KarhooUISDK.xcscheme

```diff
@@ -59,6 +59,8 @@
       buildConfiguration = "Debug"
       selectedDebuggerIdentifier = "Xcode.DebuggerFoundation.Debugger.LLDB"
       selectedLauncherIdentifier = "Xcode.DebuggerFoundation.Launcher.LLDB"
+      language = "en-US"
```

Comment:
The official local is en-GB, not en-US. That one will be added sometime in the future.